src/main.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { render as reactDomRender } from 'react-dom';
+import superagent from 'superagent';
+
+
+import '../style/main.scss';
+
+;let searchFormBoard = 'cats';
+let searchFormLimit = 5;
+
+const apiUrl = `http://reddit.com/r/${searchFormBoard}.json?limit=${searchFormLimit}`;
+
+
+class SearchForm extends React.Component {
```

Comment:
I think you're on the right path, you got stuck on figuring out how to grab the `searchFormBoard` value from the input box. I think code review may have cleared some things up for you, but please see me personally if you still need help.